eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -50,9 +50,9 @@ jobs:
     # Linux musl x64
     - ${{ if eq(parameters.platform, 'linux_musl_x64') }}:
       - ${{ if or(ne(parameters.jobParameters.isExtraPlatforms, true), eq(parameters.jobParameters.includeAllPlatforms, true)) }}:
-        - (Alpine.315.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.15-helix-amd64
+        - (Alpine.317.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.17-helix-amd64-staging
```

Comment:
If we have coverage for (latest version) 3.17 below, should we keep this at 3.15 (oldest version)?

Reply:
this is draft to force the run to check the image @am11. I did not want to force the isExtraPlatforms as tat runs many other runs.